src/lib/scripts.js

```diff
@@ -65,6 +65,14 @@ const capitalize = str => {
 };
 
 const getScriptFieldValue = (contact, texter, fieldName) => {
+  console.log("getScriptFieldValue from SCRIPTS line 68")
+  console.log("CONTACT")
+  console.log(contact)
+  console.log("TEXTER")
+  console.log(texter)
+  console.log("fieldName")
+  console.log(fieldName)
```

Comment:
This is the point where my AdminPreviewText component would fail.  After adding line 68, it wasn't until I went to the texter view, that these sections started showing real data.

If the code hadn't broken here, all my console.logs would have been only for the applyScript method below.